app/models/account.rb

```diff
@@ -54,7 +57,7 @@ class Account < ApplicationRecord
 
   # Local user validations
   with_options if: 'local?' do
-    validates :username, format: { with: /\A[a-z0-9_]+\z/i }, uniqueness: { scope: :domain, case_sensitive: false }, length: { maximum: 30 }
+    validates :username, format: { with: /\A[a-z0-9_]+\z/i }, uniqueness: { scope: :domain, case_sensitive: false }, length: { maximum: 30 }, exclusion: {in: VALID_RESERVED_WORDS}
```

Comment:
Thank you for your prompt review.

Yes, I guessed that this validation can prohibit users from creating accounts with special name.
It should be called before creation and response other name forms. Please let me clarify my understanding.